docs/server/source/cloud-deployment-templates/template-kubernetes-azure.rst

```diff
@@ -100,4 +112,32 @@ and click on the one you created
 to see all the resources in it.
 
 Next, you can :doc:`run a BigchainDB node on your new
-Kubernetes cluster <node-on-kubernetes>`.
\ No newline at end of file
+Kubernetes cluster <node-on-kubernetes>`.
+
+
+Optional: SSH to Your New Kubernetes Cluster Nodes
+--------------------------------------------------
+
+You can SSH to one of the just-deployed Kubernetes "master" nodes
+(virtual machines) using:
+
+.. code:: bash
+
+   $ ssh -i ~/.ssh/<name>.pub azureuser@<master-ip-address-or-hostname>
+
+where you can get the IP address or hostname
+of a master node from the Azure Portal.
+Note how the default username is ``azureuser``.
+
+The "agent" nodes don't get public IP addresses or hostnames,
+so you can't SSH to them *directly*,
+but you can first SSH to the master
+and then SSH to an agent from there 
+(using the *private* IP address of the agent node).
```

Comment:
private IP or hostname of the agents?
As it will be a pain to go to the portal and drill down to each node and get their IP, whereas the hostnames are listed directly at the resource groups page.